tests/lib/prepare.sh

```diff
@@ -3,7 +3,7 @@
 set -eux
 
 prepare_classic() {
-    apt install -y ${SPREAD_PATH}/../snapd_*.deb
+    dpkg -i ${SPREAD_PATH}/../snapd_*.deb || apt-get -f install -y
```

Comment:
I actually introduced a small shell function before. Turns out the one-liner looks better and serves well as a reminder that trusty is the baseline in terms of features. I'm happy to comply if you insist on a function, though :)